README.md

```diff
@@ -0,0 +1,8 @@
+# 3DSCONV
+
+3DSCONV is a utility which converts an AutoDesk 3D Studio v2.0 or v3.0 file
+into the Atari Jaguar 3D model format, used with RENDER, the 3D library developed by Atari.
+
+### Revision
+The original version, developed by Atari in 1995, has been slightly modified since by unknown source.
```

Comment:
What leads you to believe it was modified by an unknown source?  This came straight from files in the Atari source code dump thread, and was then lightly modified by me to get it to compile on modern systems.

Reply:
I didn't know you did the changes. I guess we can simply rewrite the line to include you.
"The original version, developed by Atari in 1995, has been slightly modified since by cubanismo."